### PR TITLE
@craigspaeth: More mobile

### PR DESCRIPTION
**Pull request by dzucconi**
Some more work roughing in mobile content. The copy and structuring is slightly different so it makes sense to completely break it out into it's own data file. Not a lot of sharing going on between touch and mobile versions now.

caption

### CODE REVIEW

File: src/data/mobile.json [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+        }
+      ]
+    },
+    "galleries": {
+      "panes": [
+        {
+          "headline": "Galleries uploaded over 120,000 for sale works&hellip;"
+        },
+        {
+          "headline": "&hellip;and 1,2000 shows",
```
Comment (joeyAghion) [inline src/data/mobile.json]: 1,200

File: src/data/mobile.json [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+      ]
+    },
+    "museums": {
+      "panes": [
+        {
+          "headline": "Oui!<br>12 French museums join Artsy.",
+          "backdrop": "images/lourve.jpg"
+        },
+        {
+          "headline": "Galleries and Museums uploaded over 1,2000 shows&hellip;",
```
Comment (joeyAghion) [inline src/data/mobile.json]: ditto

**Comment by gib** [comment]
so smooooth!

**Comment by craigspaeth** [comment]
Very nice! Joey pointed out some typos, feel free to merge after that :)

**Comment by craigspaeth** [comment]
I didn't notice the slight 3d effect in the .gif! Bringing back some black void action there! Looking super slick!

**Pull request merged**